### PR TITLE
Refactorings

### PR DESCRIPTION
**Pull request by dvirk**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -47,16 +47,16 @@ def load_data(self) -> None:
 
         self.map_folder = path.join(game_folder, 'maps')
 
+        self.title_font = path.join(img_folder, 'ZOMBIE.TTF')
```
Comment (mick-warehime) [inline src/main.py]: the fonts moved to the view class - this is stale

File: requirements.txt [hunk 2]
```diff
@@ -2,4 +2,5 @@ coverage==4.4.2
 flake8==3.5.0
 mypy==0.550
 pygame==1.9.3
-pytweening
\ No newline at end of file
+pytweening
```
Comment (mick-warehime) [inline requirements.txt]: versions?

Comment (dvirk) [reply requirements.txt]: I don't know what versions we're using.

File: src/sprites.py [hunk 3]
```diff
@@ -177,6 +199,11 @@ def update(self) -> None:
             self.kill()
             self.game.map_img.blit(self.game.splat, self.pos - Vector2(32, 32))
 
+    @staticmethod
+    def _target_close(target_dist: Vector2) -> bool:
+        _target_close = target_dist.length_squared() < DETECT_RADIUS ** 2
```
Comment (mick-warehime) [inline src/sprites.py]: dont need the extra variable just return instead

**Pull request merged**